Modules/Campaign/Entities/Campaign.php

```diff
@@ -3,9 +3,22 @@
 namespace Modules\Campaign\Entities;
 
 use Illuminate\Database\Eloquent\Model;
+use Illuminate\Support\Facades\DB;
 
 class Campaign extends Model {
 
-     protected $guarded = [];
+    //Declare statuses
+    const CAMPAIGN_STATUS = 'draft';
```

Comment:
Better to name status as `const STATUS_DRAFT = 'draft'`. This way we will have more flexibility for new statuses.let say if we need status active we can name it `const STATUS_ACTIVE = 'active'`